pkg/commands/app.go

```diff
@@ -151,14 +151,16 @@ func loadPluginCommands() []*cobra.Command {
 	return commands
 }
 
-func initConfig(configFile string) error {
+func initConfig(configFile string, pathChanged bool) error {
 	// Read from config
 	viper.SetConfigFile(configFile)
 	viper.SetConfigType("yaml")
 	if err := viper.ReadInConfig(); err != nil {
 		if errors.Is(err, os.ErrNotExist) {
-			log.Debug("Config file not found", log.FilePath(configFile))
-			return nil
+			if !pathChanged {
+				log.Debug("Config file not found", log.FilePath(configFile))
```

Comment:
I'd keep it at the debug level. Otherwise, it shows a message every time, which is annoying for most users.

BTW, this log is called before initializing the logger. So, it displays nothing now. We also need to fix it.
https://github.com/aquasecurity/trivy/blob/c8a7abd3b508975fcf10c254d13d1a2cd42da657/pkg/commands/app.go#L214

Reply:
@knqyf263 Understood, I'll put it back as debug.

As for the logger initialization, don't we have a chicken and egg problem here ? Or should it be initialized with defaults and then re-initialized based on the configuration ?

Reply:
>As for the logger initialization, don't we have a chicken and egg problem here ? 

Yes.

>Or should it be initialized with defaults and then re-initialized based on the configuration ?

I think you're right. We should initialize the logger twice.